samples/benchmarks/coremark/sysbuild.cmake

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+if(${images_added})
+  # same source directory is used for all targets
+  # to avoid recursive calls do not add project if remote was already added
```

Comment:
I don't really understand why this is here, what is this trying to do or work around or solve?

Reply:
Answered above

Reply:
Can you try the suggestion from https://github.com/zephyrproject-rtos/zephyr/pull/70719#issuecomment-2019940942 ?

Reply:
@zycz, could you check it please?

Reply:
Adding:

`include_guard(GLOBAL)`

works fine - it is exactly what we were dong only in neat, state of art form - thanks :)